aspnetcore/introduction-to-aspnet-core.md

```diff
@@ -4,246 +4,250 @@ author: tdykstra
+Web app | New server-side web development with MVC | <xref:tutorials/first-mvc-app/start-mvc>
+
+:::moniker-end
+
+:::moniker range=">= aspnetcore-3.0 < aspnetcore-6.0"
+
+App type | Scenario | Tutorial
+-------- | -------- | --------
+Web app | New server and client web development with Blazor - ***Recommended*** | [Build your first web app with ASP.NET Core using Blazor (Interactive Online Learn Module)](https://dotnet.microsoft.com/learn/aspnet/blazor-tutorial/intro) or [Build your first web app with Blazor (Visual Studio or Visual Studio Code)](/training/modules/build-your-first-blazor-web-app/)
+Web app | New server-side web development with Razor Pages | <xref:tutorials/razor-pages/razor-pages-start>
+Web app | New server-side web development with MVC | <xref:tutorials/first-mvc-app/start-mvc>
+Web API | Server-based data processing | <xref:tutorials/first-web-api> and [Create a web API with ASP.NET Core controllers (.NET SDK)](/training/modules/build-web-api-aspnet-core/)
+Remote Procedure Call (RPC) app | Contract-first services using Protocol Buffers | <xref:tutorials/grpc/grpc-start>
+Real-time app | Server/client bidirectional communication | <xref:tutorials/signalr>
 
 :::moniker-end
 
 :::moniker range="< aspnetcore-3.0"
 
-ASP.NET Core is a cross-platform, high-performance, [open-source](https://github.com/dotnet/aspnetcore) framework for building modern, cloud-enabled, Internet-connected apps. With ASP.NET Core, you can:
+App type | Scenario | Tutorial
+-------- | -------- | --------
+Web app | New web development | <xref:tutorials/razor-pages/razor-pages-start>
+Web app | New server-side web development with MVC | <xref:tutorials/first-mvc-app/start-mvc>
+Web API | Server-based data processing | <xref:tutorials/first-web-api> and [Create a web API with ASP.NET Core controllers (.NET SDK)](/training/modules/build-web-api-aspnet-core/)
+Real-time app | Server/client bidirectional communication | <xref:tutorials/signalr>
 
-* Build web apps and services, [Azure IoT (Internet of Things)](https://azure.microsoft.com/solutions/iot) apps, and mobile backends.
-* Use your favorite development tools on Windows, macOS, and Linux.
-* Deploy to the cloud or on-premises.
-* Run on [.NET Core or .NET Framework](/dotnet/articles/standard/choosing-core-framework-server).
+:::moniker-end
 
-## Why choose ASP.NET Core?
+The tutorials in the following table teach data access concepts.
 
-Millions of developers use or have used [ASP.NET 4.x](/aspnet/overview) to create web apps. ASP.NET Core is a redesign of ASP.NET 4.x, with architectural changes that result in a leaner, more modular framework.
+:::moniker range=">= aspnetcore-3.0"
 
-[!INCLUDE[](~/includes/benefits.md)]
+Scenario | Tutorial
+-------- | --------
+New development with Blazor | <xref:blazor/tutorials/movie-database-app/index>
+New development with Razor Pages | <xref:data/ef-rp/intro>
+New development with MVC | <xref:data/ef-mvc/intro>
```

Comment:
Just Blazor for new users - no MVC or Razor Pages. Maybe we should also have an API tutorial that includes data access concepts 🤔.

```suggestion
```